### ISSUE TITLE
What WCAG Criteria that Accessibility Insights For Android can check for

### ISSUE THREAD

**Issue by daphneyyt** [microsoft/accessibility-insights-web#6121]
Dear Accessibility Insights For Android team

We are helping our client to run WCAG scans for mobile apps using Accessibility Insights For Android. Our client is requesting for  a list of WCAG success criteria that Accessibility Insights for Android is able to check for.

**Question:** We would like to understand what are the success criteria that Accessibility Insights For Android is able to scan. 

In our initial checks, we understand that there is this list from the [Accessibility Insights For Android's website](https://accessibilityinsights.io/info-examples/android/)

From this list, we understand that the following are the criteria that can be captured:
- WCAG 1.1.1
- WCAG 1.3.1
- WCAG 1.4.3
- WCAG 1.4.11
- WCAG 2.4.3
- WCAG 2.4.4
- WCAG 2.5.5
- WCAG 3.3.2
- WCAG 4.1.2

May we know, apart from the list above, if there are any more criteria that Accessibility Insights For Android can scan?

We look forward to your response.

Thank you
Daphne

**Comment by ferBonnin** [comment]
moving this to the right repo 🙂 

@daphneyyt thanks for reaching out! Accessibility Insights has a set of automated checks and needs review checks that **partially** cover certain WCAG criteria.  Please note that Automated checks can detect some common accessibility problems such as missing or invalid properties, but most accessibility problems can only be discovered through manual testing. 

The link you added to the info-examples points to all the rules we have in Accessibility Insights for Android automated checks & Needs review. The Automated check rules and some Needs review rules are based on [axe-android](https://github.com/dequelabs/axe-android). Some Needs review rules are based on the [Accessibility Test Framework For Android](https://github.com/google/Accessibility-Test-Framework-for-Android). 

In addition to the Automated and Needs review rules, FastPass has an assisted test for TabStops that helps identify issues related to keyboard and focus.
